tests/queries/0_stateless/03130_generateSnowflakeId.sql

```diff
@@ -1,12 +1,15 @@
 -- Test SQL function 'generateSnowflakeID'
 
 SELECT bitAnd(bitShiftRight(toUInt64(generateSnowflakeID()), 63), 1) = 0; -- check first bit is zero
-
-SELECT generateSnowflakeID(1) = generateSnowflakeID(2); -- disabled common subexpression elimination --> lhs != rhs
 SELECT generateSnowflakeID() = generateSnowflakeID(1); -- same as ^^
 SELECT generateSnowflakeID(1) = generateSnowflakeID(1); -- enabled common subexpression elimination
 
-SELECT generateSnowflakeID(1, 2); -- { serverError NUMBER_OF_ARGUMENTS_DOESNT_MATCH }
+SELECT generateSnowflakeID(1) = generateSnowflakeID(2); -- disabled common subexpression elimination --> lhs != rhs
```

Comment:
Please move l. 7 back to its original position in l. 5, otherwise the comment in l. 4 is incorrect.